vgen/src/main/resources/io/gapi/vgen/java/main.snip

```diff
@@ -177,7 +183,7 @@
      *
     {@javaDocManualEdit()}
      */
-    public static final {@className} create({@settingsClassName(service)} settings) throws IOException {
+    public static final {@className} create({@settingsClassName(service)} settings) throws IOException, IllegalStateException {
```

Comment:
You don't have to declare that you throw IllegalStateException because it is a RuntimeException (also applies to the GAX PR)